diag_manager/fms_diag_field_object.F90

```diff
@@ -406,6 +408,24 @@ subroutine set_vartype(objin , var)
  end select
 end subroutine set_vartype
 
+!> @brief Sets the send data time
+subroutine set_send_data_time (this, time)
```

Comment:
Are these wrappers over the other set_send_data_time routines?  This seems convoluted.

Reply:
This is the same thing that was done with the file and yaml object:
https://github.com/NOAA-GFDL/FMS/blob/dmUpdate/diag_manager/fms_diag_file_object.F90#L125-L140

It was done like this for convenience.

Reply:
Sure, but the description is wrong.  This doesn't set anything.  It calls `input_data_buffer%set_send_data_time` which does the setting.